docs/src/use/command-line-interface.md

```diff
@@ -466,6 +466,8 @@ This option allows you to specify a warning threshold, which can be used to forc
 
 Normally, if ESLint runs and finds no errors (only warnings), it exits with a success exit status. However, if `--max-warnings` is specified and the total warning count is greater than the specified threshold, ESLint exits with an error status.
 
+When used alongside `--quiet`, this will cause rules marked as warn to still be run, but not reported.
```

Comment:
Let's emphasize this point.

```suggestion
::: important
When used alongside `--quiet`, this will cause rules marked as warn to still be run, but not reported.
:::
```